Sources/Commands/SwiftPackageTool.swift

```diff
@@ -30,44 +30,44 @@ import func POSIX.chdir
 extension PackageToolOptions: XcodeprojOptions {}
 
 private enum Mode: Argument, Equatable, CustomStringConvertible {
-    case Init(InitMode)
+    case initPackage
     case doctor
-    case showDependencies(ShowDependenciesMode)
+    case showDependencies
     case fetch
     case update
     case usage
     case version
-    case generateXcodeproj(String?)
-    case dumpPackage(String?)
+    case generateXcodeproj
+    case dumpPackage
 
     init?(argument: String, pop: () -> String?) throws {
         switch argument {
         case "init", "initialize":
-            self = try .Init(InitMode(pop()))
+            self = .initPackage
         case "doctor":
             self = .doctor
-        case "show-dependencies", "-D":
-            self = try .showDependencies(ShowDependenciesMode(pop()))
+        case "show-dependencies", "show-deps":
+            self = .showDependencies
         case "fetch":
             self = .fetch
         case "update":
             self = .update
         case "help", "usage", "--help", "-h":
             self = .usage
-        case "version":
+        case "version", "--version":
```

Comment:
I'm actually fine with just an option, as I believe that to be the most standard. The way I view the mental model is that arguments like `--version` are options to the "swift-package" command, while things like `show-dependencies` are subcommands (that may inherit common options, and may have options of their own).